docs/configuration/registration/zgw.rst

```diff
@@ -105,6 +105,7 @@ Configuration
    * **Zaaktype**: Select the default Zaaktype to be used to create the Zaak. *For example:* ``https://example.com/catalogi/api/v1/zaaktypen/1/``
    * **Informatieobjecttype**: Fill in the URL of the Informatieobjecttype to be used to create the Document. *For example:* ``https://example.com/catalogi/api/v1/informatieobjecttypen/1/``
    * **Organisatie RSIN**: Fill the RSIN to be referred to in the created objects. *For example:* ``123456789``
+   * **Medewerker roltype**: Fill in the URL of the Roltype to be used when creating a role for an employee filling in a form for a citizen or company. For example: ``https://example.com/catalogi/api/v1/roltypen/2/``
```

Comment:
Let's start using user-friendly fields/parameters for this - every time you get a new zaaktype version, the roltype URLs change and this is a maintenance burden. Instead, they should be able to configure the roltype `omschrijving` and our code should match this within the zaaktype.